CUJExecutor/dataAggregation/AggregateExecuteCuj.java

```diff
@@ -0,0 +1,173 @@
+package dataAggregation;
+import java.io.File;  // Import the File class
+import java.io.FileNotFoundException;  // Import this class to handle errors
+import java.util.Scanner; // Import the Scanner class to read text files
+import java.util.*;
+import java.io.*;
+
+class AggregateExecuteCuj {
+	
+
+	/**
+	 * parses the test given by myReader, which is formatted as specified by the JAVADOC on the main function of this class.
+	 * Adds the parsed 
+	 *  - durations to the actionDurationsList, 
+	 *  - clipInfo object to the totalDurationsToClipinfo map.
+	 *
+	 */
+	public static void parseInputFile (Scanner myReader, List<List<Integer>> actionDurations,  Map<Integer, Utility.clipInfo> totalDurationToClipInfo) {
```

Comment:
1. should use a Java object to represent an actionDuration
2. class name `clipInfo` must be capitalized.